src/Microsoft.Windows.CsWin32/Generator.cs

```diff
@@ -82,6 +82,20 @@ public class Generator : IDisposable
 
         private const string SimpleFileNameAnnotation = "SimpleFileName";
         private const string OriginalDelegateAnnotation = "OriginalDelegate";
+        private static readonly SyntaxTriviaList InlineArrayUnsafeAsSpanComment = ParseLeadingTrivia(@"/// <summary>
+/// Gets this inline array as a span.
+/// </summary>
+/// <remarks>
+/// Important: When this struct is on the stack, do not let the returned span outlive the stack frame that defines it.
```

Comment:
I sometimes use the '⚠' character in XML docs.

Reply:
Sounds good.